src/Factory/ProjectionManagerFactory.php

```diff
@@ -68,14 +68,24 @@ public function addResolver(string $name, Closure $resolver): void
     public function managerFor(string $projection): ProjectionManager
     {
         foreach ($this->app->make('config')->get('event_store.projection_managers') as $name => $config) {
-            if (in_array($projection, array_keys($config['projections']))) {
+            if (array_key_exists($projection, $config['projections'])) {
                 return $this->make($name);
             }
         }
 
         throw new RuntimeException('No projection manager for projection ' . $projection);
     }
 
+    public function optionsFor(string $projection) {
```

Comment:
Can you add the `array` return type here and drop the opening bracket to the next line to maintain consistency.
```suggestion
    public function optionsFor(string $projection): array
    {
```

Reply:
Done 👍